lib/hammer_cli/context.rb

```diff
@@ -3,9 +3,10 @@
 module HammerCLI
 
   def self.context
-    {
+    @context ||= {
       :defaults => HammerCLI.defaults,
-      :is_tty? => HammerCLI.tty?
+      :is_tty? => HammerCLI.tty?,
+      :api_connection => HammerCLI::Connection.new(Logging.logger['Connection'])
```

Comment:
Would it make sense to move this to command initializer and add just the attributes missing? This way you could provide only part of the context to the command and the rest would be replaced with the defaults. The context would also be accessible with `@context`?